src/com/theironyard/novauc/ATM.java

```diff
@@ -0,0 +1,111 @@
+package com.theironyard.novauc;
+
+import java.util.Scanner;
+
+public class ATM{
+
+    public static void main (String[] args) throws Exception {
+        // write your code here
+
+        User customer = new User(0, 75, 25);
+
```

Comment:
There is an error on line 10.  You do not have a constructor with only three arguments.  So you need to give that constructor another parameter.